examples/typescript/tsconfig.json

```diff
@@ -1,7 +1,12 @@
 {
     "extends": "../../tsconfig.json",
+    "compilerOptions": {
```

Comment:
These changes allow top level `await` and the use of `@kubernetes/client-node` in the TypeScript examples.